class.jetpack.php

```diff
@@ -7364,6 +7364,12 @@ public static function load_jetpack_gutenberg() {
 			$version
 		);
 
+		$script = sprintf(
+			'var JETPACK_BLOCK_ASSETS_BASE_URL = %s;',
+			wp_json_encode( plugins_url( '_inc/blocks/', JETPACK__PLUGIN_FILE ) )
+		);
+		wp_add_inline_script( 'jetpack-blocks-editor', $script, 'before' );
```

Comment:
Could we use `wp_localize_script()` instead of `wp_add_inline_script()` here? The nice thing about it is that we won't need to have `var` or any `sprintf` or any manual json encoding. 

If you'd like to see an example usage, you can take a look at how the initial state of the Jetpack React app is passed to the app: https://github.com/Automattic/jetpack/blob/b578d604cb0aba90fe47fc848259c1e358d2de7c/_inc/lib/admin-pages/class.jetpack-react-page.php#L232

Reply:
That's better, I've applied the change.